changelog/unreleased/change-ocselect-label

```diff
@@ -0,0 +1,8 @@
+Change: Label for OcSelect
+
+We've added a configurable `<label>` for OcSelect accessible via the `label` property.
+This shadows the `label` property of `vue-select`. Hence we introduced the `optionLabel`
+property on OcSelect which maps to the `label` property of `vue-select`.
+
+https://github.com/owncloud/owncloud-design-system/pull/1570/
```

Comment:
Please also add the link to this PR here, i.e. https://github.com/owncloud/owncloud-design-system/pull/1633